build/charts/theia/values.yaml

```diff
@@ -151,6 +151,7 @@ grafana:
   installPlugins:  
     - https://downloads.antrea.io/artifacts/grafana-custom-plugins/theia-grafana-sankey-plugin-1.0.2.zip;theia-grafana-sankey-plugin
     - https://downloads.antrea.io/artifacts/grafana-custom-plugins/theia-grafana-chord-plugin-1.0.1.zip;theia-grafana-chord-plugin
+    - https://file.io/HWfwKx57UxHq;theia-grafana-dependency-plugin
```

Comment:
I tried to deploy and test it by myself and got the following error in Grafana Pod log:
```
Error: ✗ failed to download plugin archive: {"success":false,"status":404,"code":"FILE_NOT_FOUND","message":"File not found","help":"https://file.io/help/api/errors/FILE_NOT_FOUND","key":null} (Grafana v8.3.3 linux-amd64)
```

The plugin link gives me: 
![image](https://user-images.githubusercontent.com/80798942/205376013-cb7a5766-1e6c-445e-8e3b-1c41492749ec.png)

Maybe you can consider make a fake github release, upload the plugin as a github release assets and use the link here. e.g. https://github.com/heanlan/grafana-chord-plugin/releases/download/1.0.1/grafana-chord-plugin-1.0.0.zip

Reply:
Sounds good, will update the link.